tests/testthat/test-slice-chop.R

```diff
@@ -0,0 +1,423 @@
+# vec_chop ----------------------------------------------------------------
```

Comment:
```suggestion
```

Reply:
If you create the new files after the PR is merged (or create right away but move the old tests after merge), it's easier to review if there is any new tests or changes.